src/routes/api/api-hash-post.ts

```diff
@@ -0,0 +1,22 @@
+import { Request, Response } from "express";
+import { createHashWithSharedSecret } from "~/src/util/encryption";
+
+export const ApiHashPost = async (req: Request, res: Response): Promise<void> => {
+
+	const { data } = req.body;
+
+	if (!data) {
+		res.status(400)
+			.json({
+				message: "Please provide a value to be hashed."
+			});
+		return;
+	}
+
+	const hashedValue = createHashWithSharedSecret(data);
+
+	res.json({
```

Comment:
Could just return the hashed value and nothing else.... mainly being used from a terminal so was nice to get a confirmation of what i was hashing.... maybe a security issue though pairing them up like that?

Reply:
I'm not sure I understand the point of this API?  We call this first API to be able to get data from other APIs?

We don't need to be this drastic.  Our API can still take jirahost to return relevant data.